packages/analytics-js/src/components/core/Analytics.ts

```diff
@@ -613,14 +577,15 @@ class Analytics implements IAnalytics {
 
   group(payload: GroupCallOptions) {
     const type = 'group';
-    this.errorHandler.leaveBreadcrumb(`New ${type} event`);
-    state.metrics.triggered.value += 1;
 
     if (!state.lifecycle.loaded.value) {
       state.eventBuffer.toBeProcessedArray.value.push([type, payload]);
       return;
     }
 
+    this.errorHandler.leaveBreadcrumb(`New ${type} event`);
```

Comment:
Now that the call for the leaveBreadcrumb and metrics count is after the case for state.lifecycle.loaded.value, we loose the counting of the metod calls. If I recollect the eventBuffer does not process its items via the same methods.

why we did this change?

Reply:
The event buffer processes the items via the same methods. Hence, I moved it after the buffer logic to avoid duplicate breadcrumbs and metrics.

Reply:
perfect